lib/shopify-cli/shopifolk.rb

```diff
@@ -0,0 +1,66 @@
+require 'fileutils'
+
+module ShopifyCli
+  ##
+  # ShopifyCli::Shopifolk contains the logic to determine if the user is a Shopify employee
+  #
+  # The Shopify Feature flags will persist between runs so if the flag is enabled or disabled,
+  # it will still be in that same state until the next cli invocation.
+  class Shopifolk
+    GCLOUD_CONFIG_PATH = '~/.config/gcloud/configurations/config_default'
+    SECTION = 'core'
+    FEATURE_NAME = 'shopifolk'
+    def self.check
+      ##
+      # will return if the user is a Shopify employee
```

Comment:
Since it's not definitive, I think we should say something like 'if the user appears to be a Shopify employee, based on several heuristics'